cmd/nerdctl/image_encrypt_linux_test.go

```diff
@@ -80,7 +80,6 @@ func TestImageEncryptJWE(t *testing.T) {
 	encryptImageRef := fmt.Sprintf("127.0.0.1:%d/%s:encrypted", reg.ListenPort, tID)
 	defer base.Cmd("rmi", encryptImageRef).Run()
 	base.Cmd("image", "encrypt", "--recipient=jwe:"+keyPair.pub, testutil.CommonImage, encryptImageRef).AssertOK()
-	base.Cmd("image", "inspect", "--mode=native", "--format={{len .Index.Manifests}}", encryptImageRef).AssertOutExactly("1\n")
```

Comment:
We still have to assert the len of `Manifests`

Reply:
I can't find Manifests just find Manifest

```
[root@LIN-FFF47298CDA nerdctl]# nerdctl  inspect  --mode=native   test:encrypted
[
    {
        "Image": {
            "Name": "docker.io/library/test:encrypted",
            "Labels": null,
            "Target": {
                "mediaType": "application/vnd.oci.image.manifest.v1+json",
                "digest": "sha256:da12013fd309ef8732aae4f22cf2a14938b3219f761b8943e78b16d87cd181d5",
                "size": 1764
            },
            "CreatedAt": "2022-02-17T06:50:37.280484768Z",
            "UpdatedAt": "2022-02-17T06:50:37.280484768Z"
        },
        "ManifestDesc": {
            "mediaType": "application/vnd.oci.image.manifest.v1+json",
            "digest": "sha256:da12013fd309ef8732aae4f22cf2a14938b3219f761b8943e78b16d87cd181d5",
            "size": 1764
        },
        "Manifest": {
            "schemaVersion": 2,
            "config": {
                "mediaType": "application/vnd.docker.container.image.v1+json",
                "digest": "sha256:8f1471b5fad2d23e1f8cb157bad6e583b21e9b388e932c355c57817294cddaa4",
                "size": 2032
            },
            "layers": [
                {
                    "mediaType": "application/vnd.oci.image.layer.v1.tar+gzip+encrypted",
                    "digest": "sha256:55acb6e82c93f8e2388147b4233f7af4458c7606af4f95073751cd43cba3f3aa",
                    "size": 249517056,
                    "annotations": {
                        "org.opencontainers.image.enc.keys.jwe": "eyJwcm90ZWN0ZWQiOiJleUpoYkdjaU9pSlNVMEV0VDBGRlVDSXNJbVZ1WXlJNklrRXlOVFpIUTAwaWZRIiwiZW5jcnlwdGVkX2tleSI6IkdJQ1p3QXRXTXNhZzBvTDhpbV9OdHRqZ0p4VGFVX3ZGN3Z6NUVRU0lfcGVaM0hKZWxzbVVzbVpEZHAwLWdKSTRjRzNuUzJPTmlGRHZEY0h1ZWtya01PTmVVcENsZjJHdS1BYkZxU1FTRW5QWThuTkMtVlpPRUF6eGRaaU9HSnlLSGFlWmc1OUpkb09RNXNGUVdBWV82NTZoQ2xkeGJ2LWJsbEZPTGpGQlEtS2lWZWRIZTRZYkdwQjNSc05PakNrckVza1BfX1doQ09CRVNYR0pTZFRnYTFDa3FOczFyd2VpYk5DZTRseEwwVElHUWZCWEh2VzhHbUhxTTFWazYwTDBTbHhpMWhld2pFdEppZlNtcmZ5UTFBdEVxVE13WHdZZF9OM0I0SmlucGhvSHdubDF1QjhvS2V5bTFXNzdmaHFsVUpfdUZPdERmTGFPS0E2a21LaG1KdyIsIml2IjoiQmducVo5YVBNV3B0Qk5JXyIsImNpcGhlcnRleHQiOiJWb3pqU2h5TmtCZTEweU9WeG9tWTBJdlZoOE5SZDNLVGx1ZkJuNVRteHE2OVNHZTVBSVJ6ZjNVZWhnd2ExdWwyeU03TkF1bTZmaDBNalpaTGN5aVRiWW1mWVlEWGY4SFRLU0xUM2dFby05VVZfNlZzSU9SNFZOc1czMHNNclk1Z3BZMC01dmhtOEpZUUlVWGxpaW5GUkx3MjhOZFlvNXNrOUoxWVFzcGNZMlNQR2NQOEx1QkRiQjFuMjFKcVBZZVZBWDUzUmpRMGVLdUpCOHNSTTFlSFRLTDRaUzJEcnV5VGNwWUVhMWp6VmR6SFVybTVub1llZDZzeFhwTTVWWWZUOHciLCJ0YWciOiJiNkpPNDJlT014NWlfeXNpUE91cE1nIn0=",
                        "org.opencontainers.image.enc.pubopts": "eyJjaXBoZXIiOiJBRVNfMjU2X0NUUl9ITUFDX1NIQTI1NiIsImhtYWMiOiIxRG5uSkU4TmZkZ2ErWGtKSXdvdnFZd3NCYjZITW5qZzM4SVNYTTZTYkRBPSIsImNpcGhlcm9wdGlvbnMiOnt9fQ=="
                    }
                }
            ]
        },
        "ImageConfigDesc": {
            "mediaType": "application/vnd.docker.container.image.v1+json",
            "digest": "sha256:8f1471b5fad2d23e1f8cb157bad6e583b21e9b388e932c355c57817294cddaa4",
            "size": 2032
        },
        "ImageConfig": {
            "created": "2020-02-21T02:40:48.303555757Z",
            "architecture": "amd64",
            "os": "linux",
            "config": {
                "Env": [
                    "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
                    "DISTTAG=f30-updates-candidatecontainer",
                    "FGC=f30-updates-candidate",
                    "FBR=f30-updates-candidate"
                ],
                "Cmd": [
                    "/bin/bash"
                ],
                "Labels": {
                    "maintainer": "Clement Verna \u003ccverna@fedoraproject.org\u003e"
                }
            },
            "rootfs": {
                "type": "layers",
                "diff_ids": [
                    "sha256:0e9db48c579d098b58a078ee45fc6490b2c297d677ee63a3da3ae032c59eb4d6"
                ]
            },
            "history": [
                {
                    "created": "2019-01-16T21:21:55.569693599Z",
                    "created_by": "/bin/sh -c #(nop)  LABEL maintainer=Clement Verna \u003ccverna@fedoraproject.org\u003e",
                    "empty_layer": true
                },
                {
                    "created": "2019-09-27T21:20:49.838093153Z",
                    "created_by": "/bin/sh -c #(nop)  ENV DISTTAG=f30-updates-candidatecontainer FGC=f30-updates-candidate FBR=f30-updates-candidate",
                    "empty_layer": true
                },
                {
                    "created": "2020-02-21T02:40:47.885678191Z",
                    "created_by": "/bin/sh -c #(nop) ADD file:7eccfd20ecf40bd0c40fe783141bcea78524531160e7a8fc5f77993aa37507d7 in / "
                },
                {
                    "created": "2020-02-21T02:40:48.303555757Z",
                    "created_by": "/bin/sh -c #(nop)  CMD [\"/bin/bash\"]",
                    "empty_layer": true
                }
            ]
        }
    }
]

```